### PR TITLE
Questionsroute

### PR DESCRIPTION
**Pull request by brennachen**
Sprint 2:
- used list of questions from AirTable to fill questions list on Questions Page
- centered 'Questions' title and replaced logo with counter for questions answered (doesn't work yet)
- onPress color for questions is now gray

image

### CODE REVIEW

File: src/screens/Home/HomeScreen/HomeScreen.tsx [hunk 1]
```diff
@@ -12,48 +12,48 @@ import {
...
         <View style={styles.banner}>
           <Text style={styles.bannertext}>QUESTIONS</Text>
-          <Image source={logo} style={styles.bannerlogo} />
+          <View style = {styles.counter}>
+              <View style = {styles.numberCounter}>
+                <Text style={styles.number}>0</Text>
+              </View>
+              <Text style={styles.answered}>answered</Text>
+            </View>
         </View>
```
Comment (apetranik) [inline src/screens/Home/HomeScreen/HomeScreen.tsx]: This header section is actually in the true header of the screen in the figma. This is set with react navigation options. Below your function you can write something like this. (you may have to fiddle w/ the styling)

```js
HomeScreen.navigationOptions = {
  header: <Text style={styles.bannertext}>QUESTIONS</Text>,
  headerRight: 
    <View style = {styles.counter}>
      <View style = {styles.numberCounter}>
        <Text style={styles.number}>0</Text>
      </View>
      <Text style={styles.answered}>answered</Text>
    </View>
}
```

**Comment by brennachen** [comment]
Hi Aliya,

Thanks for the feedback! I'll be sure to implement them over the next few
days.

Thanks,

Brenna

File: src/screens/Home/HomeScreen/HomeScreen.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, {useState} from 'react'
+import React, { useState, useEffect } from 'react'
 import logo from '../../../../assets/images/gladeo_logo.png'
```
[the remark this answers is not in the document]
Comment (apetranik) [reply src/screens/Home/HomeScreen/HomeScreen.tsx]: bump

File: src/screens/Home/HomeScreen/HomeScreen.tsx (hunk 1 above)
Comment (apetranik) [reply src/screens/Home/HomeScreen/HomeScreen.tsx]: bump

**Comment by brennachen** [comment]
Hi Aliya,

Yes! I made some more changes just now to add in the navigation bar (the
counter spacing is still a little funky, I'll try to figure out why it's so
close to the right side and how to fix that). I also was unsure what you
meant on deleting the unused import, as my code seems to use both useEffect
and useState.

From now on, I can also thumbs up the PR comments. Thank you for your
feedback and for being so patient - I'm sorry it ended up taking so long!

Sincerely,

Brenna

**Comment by apetranik** [comment]
Yea you can ignore the unused import comment you're right. And I'll help you out with the spacing. I just pushed to your branch w/ a small style fix so it should be good to go!

**Pull request merged**